packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -42,4 +68,71 @@ describe('Error', () => {
         // Assert
         expect(description.text()).toMatchSnapshot();
     });
+
+    describe('methods', () => {
+        describe('`redirectToMenu` ::', () => {
+            const oldWindowLocation = window.location;
+
+            beforeAll(() => {
+                delete window.location;
+
+                window.location = Object.defineProperties(
+                    {},
```

Comment:
Is this right?

Reply:
Yes, needs two arguments here. First is the object second is the properties.

Reply:
Thanks 🙂